<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Professionnel</class>
 <widget class="QDialog" name="Professionnel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1250</width>
    <height>850</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1250</width>
    <height>850</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1250</width>
    <height>850</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="toolTip">
   <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;GÃ©rer ici les Professionnels&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/*background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:1 rgba(0, 255, 150, 255));*/</string>
  </property>
  <widget class="QTableView" name="tableView_client">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>10</y>
     <width>700</width>
     <height>680</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>700</width>
     <height>680</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>700</width>
     <height>680</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ListerPro">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>690</y>
     <width>221</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>370</width>
     <height>340</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Lister tous les Professionnels</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Retour">
   <property name="geometry">
    <rect>
     <x>1090</x>
     <y>690</y>
     <width>121</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Retour au Menu</string>
   </property>
  </widget>
  <widget class="QLabel" name="statut">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>910</y>
     <width>981</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>-10</y>
     <width>481</width>
     <height>161</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="label_3">
      <property name="styleSheet">
       <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(255, 255, 255, 255));
color: rgb(0, 0, 0);
font: 75 8pt &quot;MS Shell Dlg 2&quot;;


</string>
      </property>
      <property name="text">
       <string>                          CREER UN NOUVEAU PROFESSIONNEL</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="styleSheet">
         <string notr="true">font: 75 10pt &quot;MS Shell Dlg 2&quot;;
text-decoration: underline;
color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string>Nom:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="styleSheet">
         <string notr="true">font: 75 10pt &quot;MS Shell Dlg 2&quot;;
color: rgb(0, 0, 0);
text-decoration: underline;</string>
        </property>
        <property name="text">
         <string>Prenom:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="PrenomNouvPro">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="nomNouvPro"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="addNewPro">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>AJOUTER</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>190</y>
     <width>491</width>
     <height>161</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QLabel" name="label_4">
      <property name="styleSheet">
       <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(255, 255, 255, 255));
color: rgb(0, 0, 0);
font: 75 8pt &quot;MS Shell Dlg 2&quot;;


</string>
      </property>
      <property name="text">
       <string>                AFFICHER LES INFO D'UN PRO OU LE SUPPRIMER</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QFormLayout" name="formLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label_5">
        <property name="styleSheet">
         <string notr="true">font: 75 10pt &quot;MS Shell Dlg 2&quot;;
text-decoration: underline;
color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string>Professionnel:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="comboBox_loadDataPro">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="consulterInfoPro">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>AFFICHER INFORMATIONS</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="SuppPro">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>SUPPRIMER PROFESSIONNEL</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_4">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>390</y>
     <width>491</width>
     <height>171</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QLabel" name="label_6">
      <property name="styleSheet">
       <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(255, 255, 255, 255));
color: rgb(0, 0, 0);
font: 75 8pt &quot;MS Shell Dlg 2&quot;;


</string>
      </property>
      <property name="text">
       <string>            METTRE A JOUR LES INFORMATIONS D'UN PROFESSIONNEL</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QFormLayout" name="formLayout_3">
      <item row="0" column="0">
       <widget class="QLabel" name="label_7">
        <property name="styleSheet">
         <string notr="true">font: 75 10pt &quot;MS Shell Dlg 2&quot;;
text-decoration: underline;
color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string>Professionnel:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="comboBox_majInfoPro">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="nouveauNomLabel">
        <property name="styleSheet">
         <string notr="true">font: 75 10pt &quot;MS Shell Dlg 2&quot;;
text-decoration: underline;
color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string>Nouveau Nom:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="NouvNomMajInfoPro">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="nouveauPrenomLabel">
        <property name="styleSheet">
         <string notr="true">font: 75 10pt &quot;MS Shell Dlg 2&quot;;
text-decoration: underline;
color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string>Nouveau Prenom:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="NouvPrenomMajInfoPro"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="majInfoPro">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>MISE A JOUR</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_5">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>600</y>
     <width>481</width>
     <height>201</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <widget class="QLabel" name="label_10">
      <property name="styleSheet">
       <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(255, 255, 255, 255));
color: rgb(0, 0, 0);
font: 75 8pt &quot;MS Shell Dlg 2&quot;;


</string>
      </property>
      <property name="text">
       <string>                AJOUTER/SUPRIMER UNE COMPETENCE </string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QFormLayout" name="formLayout_5">
      <item row="0" column="0">
       <widget class="QLabel" name="label_11">
        <property name="styleSheet">
         <string notr="true">font: 75 10pt &quot;MS Shell Dlg 2&quot;;
text-decoration: underline;
color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string>Professionnel:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="comboBox_ProAddSuppCpt">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="nouveauNomLabel_3">
        <property name="styleSheet">
         <string notr="true">font: 75 10pt &quot;MS Shell Dlg 2&quot;;
text-decoration: underline;
color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string>Competence Ã  ajouter:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="cptToAdd">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="nouveauPrenomLabel_3">
        <property name="styleSheet">
         <string notr="true">font: 75 10pt &quot;MS Shell Dlg 2&quot;;
text-decoration: underline;
color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string>Competence Ã  supprimer</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="comboBox_CptToSup">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="AddCpt">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>AJOUTER</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="SupCpt">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>SUPPRIMER</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
